spec/lib/tasks/db_data_migrate_rake_spec.rb

```diff
@@ -141,5 +141,66 @@
         expect(openid_authentication_service).to be_a OpenidAuthenticationService
       }
     end
+
+    shared_examples 'a consistency migration' do |prep_method_sym|
+      let(:record_class) { record.class }
+      before do
+        expect(record).to be_persisted
+      end
+      context 'record is consistent' do
+        let(:prep_method) { send(prep_method_sym) }
+        before do
+          prep_method.call
```

Comment:
Using a `Proc` and having to use `.call` doesn't really fit any of our existing patterns. I believe you could get the same result without a `Proc` and by replacing the above with: `expect { proc_method }.not_to raise_error`

Which gives the added bonus of trapping possible exceptions when `prep_method` is executed.